content/html/glossaries/semantic-html/aside.md

```diff
@@ -0,0 +1,48 @@
+---
+Title: "<aside>"
+Subjects:
+  - "Web Development"
+  - "Web Design"
+Tags:
+  - "Semantic"
+  - "Tags"
+Catalog Content:
+  - "https://www.codecademy.com/learn/learn-html"
+  - "https://www.codecademy.com/learn/paths/web-development"
+---
+
+## Definition 
+
+The `<aside>` semantic tag represents a part of a page which is relevant to the current content, but not a part of it directly. It can be used for related links, for clarifying a statement from the current article, or even for advertising meant for the current page. 
+
+## Syntax
+
+```html
+<aside>
+  <!-- Article content goes inside the tag. -->
+</aside>
+``` 
+
+`<aside>` is a tag that wraps around a block of HTML giving it semantic meaning. It has no special attributes, and by default will act similarly to a `<div>`.
+
+## Example
+
+Suppose the current page wants to feature a definition of an important term without having it be a part of the text the term appears in:
+
+```html
```

Comment:
Should we add a full file? With:

```html
<html>
  <head>
  </head>
  <body>
     Move the code here?
  </body>
</html>
```

Reply:
Let's also take out the `<em>`s to not distract.

Reply:
Done